File: android/src/main/java/com/getcapacitor/community/admob/models/FullScreenAdEventName.java
package com.getcapacitor.community.admob.models;

public enum FullScreenAdEventName {
    onAdLoaded("onAdLoaded"),
    onAdFailedToLoad("onAdFailedToLoad"),
    adDidPresentFullScreenContent("adDidPresentFullScreenContent"),
    didFailToPresentFullScreenContentWithError("didFailToPresentFullScreenContentWithError"),

    /**
     * Follow iOS Event Name
     * https://developers.google.com/admob/ios/api/reference/Protocols/GADFullScreenContentDelegate#-addidpresentfullscreencontent:
     */
    adDidDismissFullScreenContent("adDidDismissFullScreenContent");

    private String event;

    FullScreenAdEventName(String event) {
        this.event = event;
    }
}

File: ios/Sources/AdMobPlugin/Models/FullScreenAdEventName.swift
public enum FullScreenAdEventName: String {
    case onAdLoaded = "onAdLoaded"
    case onAdFailedToLoad = "onAdFailedToLoad"
    case adDidPresentFullScreenContent = "adDidPresentFullScreenContent"
    case didFailToPresentFullScreenContentWithError = "didFailToPresentFullScreenContentWithError"

    /**
     * Follow iOS Event Name
     * https://developers.google.com/admob/ios/api/reference/Protocols/GADFullScreenContentDelegate#-addidpresentfullscreencontent:
     */
    case adDidDismissFullScreenContent = "adDidDismissFullScreenContent"
}
